boards/nxp/frdm_mcxa156/board.c

```diff
@@ -15,7 +15,7 @@
 /* System clock frequency. */
 extern uint32_t SystemCoreClock;
 
-static int frdm_mcxa156_init(void)
+void board_early_init_hook(void)
```

Comment:
To enable the calling of this hook, we need enable it somewhere like this:
config BOARD_EARLY_INIT_HOOK
	default y

Reply:
@Raymond0225 it is being enabled in the board Kconfig